### ISSUE TITLE
NTR: Thymine high in CSF (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8465]
New term request Thymine high in CSF (elthat_111121043005): Cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000207
- label: Elevated CSF thymine concentration
- parents: [Abnormal CSF metabolite concentration (HP:0025454)](https://hpo.jax.org/app/browse/term/HP:0025454)
- pmid: 
- def: The concentration of thymine in the cerebrospinal fluid (CSF) is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8465
- synonyms: Thymine high in CSF(NONE)
